### PR TITLE
ipv64: remove unused option

### PR DESCRIPTION
**Pull request by ldez**
`IPV64_SEQUENCE_INTERVAL` is unused.

### CODE REVIEW

File: providers/dns/ipv64/ipv64.go [hunk 1]
```diff
@@ -24,15 +24,13 @@ const (
...
 	EnvHTTPTimeout        = envNamespace + "HTTP_TIMEOUT"
-	EnvSequenceInterval   = envNamespace + "SEQUENCE_INTERVAL"
 )
 
 // Config is used to configure the creation of the DNSProvider.
 type Config struct {
 	APIKey             string
 	PropagationTimeout time.Duration
 	PollingInterval    time.Duration
-	SequenceInterval   time.Duration
```
Comment (dmke) [inline providers/dns/ipv64/ipv64.go:35]: Doesn't this change make it a breaking change? `Config` is exported, so removing fields could affect Lego-as-library users.

For now, I'd just add a comment and mark it for removal in v5, e.g.

```diff
-	SequenceInterval   time.Duration
+ 	SequenceInterval   time.Duration // Deprecated and not used. Will be removed in Lego v5.
```

**Review by dmke — CHANGES_REQUESTED** [review]

File: providers/dns/ipv64/ipv64.go (hunk 1 above)
Comment (ldez) [reply providers/dns/ipv64/ipv64.go:35]: There is no breaking change because the option is not used and the option was introduced in v4.13.

Comment (ldez) [reply providers/dns/ipv64/ipv64.go:35]: If we keep and deprecate the field `SequenceInterval` we also have to keep and deprecate the env var name (constant).

Comment (dmke) [reply providers/dns/ipv64/ipv64.go:35]: Hm, true... So do we remove it (and deal with the fallout later, if there's any), or do we keep them with a deprecation note?

**Review by dmke — APPROVED** [review]

File: providers/dns/ipv64/ipv64.go (hunk 1 above)
Comment (ldez) [reply providers/dns/ipv64/ipv64.go:35]: We can keep them, it's better from the point of view of the project.

**Pull request merged**